dispatcher_plugin_multimessenger/dataserver_dispatcher.py

```diff
@@ -76,7 +77,7 @@ def run_query(self,
         if param_dict is None:
             param_dict=self.param_dict   
         
-        if run_asynch:
+        if run_asynch and not self.no_asynch:
```

Comment:
why two variables for `asynch` running mode?
do they serve different purposes?

Reply:
Here I use the property `no_asynch` to be able to temporarily disable asynchronicity of this Instrument in MMProposalQuery.
In general, asynchronous mode is set for the whole instrument and the parameter `run_asynch` is passed to run_query() based on this setting

I'm trying now to find a cleaner way to forcibly make the query synchronous

Reply:
isn't the handling of the `run_asynch` value at frontend level sufficient? 
are there situations where we have to force the asynch execution to be interrupted and cannot be done via the frontend?

Reply:
Oops, really. I've totally forgot that it could be just passed in the request. It simplifies everything. Thanks